Makefile

```diff
@@ -41,7 +41,7 @@ check_integration_test:
 
 .PHONY: rustls_unit_test
 rustls_unit_test:
-	cargo +$$RUST_VERSION test --all -v --no-default-features --features=rustls
+	(cd rusoto/core && cargo +$$RUST_VERSION test --no-default-features --features=rustls)
```

Comment:
I changed this line to `(cd rusoto/core && cargo +$$RUST_VERSION test --no-default-features --features=rustls && cd ../services && cargo +$$RUST_VERSION test --no-default-features --features=rustls)` and the Makefile isn't working as I expected it to: I get `error: --features is not allowed in the root of a virtual workspace` but in `rusoto/services` I can run `cargo test --no-default-features --features=rustls` without a problem. 🤔